Packs/SekoiaXDR/ReleaseNotes/1_1_10.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### Sekoia XDR
+
+- Fix some issues on sekoia http_request command
```

Comment:
please write the note in this format
- fixed an issue with the *** http_request_command*** command where <one sentence about the  problem>